docs/en/api/adc.rst

```diff
@@ -52,7 +52,8 @@ This function will return analog value in millivolts (calibrated).
 analogReadResolution
 ^^^^^^^^^^^^^^^^^^^^
 
-This function is used to set the resolution of ``analogRead`` return value. Default is 12 bits (range from 0 to 4095).
+This function is used to set the resolution of ``analogRead`` return value. Default is 12 bits (range from 0 to 4095)
```

Comment:
This is only for the chip revision v0.0. For v1.0, the resolution is 12 bits.

Please add a reference to the [ADC-112 Bit 1 of SAR ADC Does Not Flip errata](https://docs.espressif.com/projects/esp-chip-errata/en/latest/esp32s2/03-errata-description/esp32s2/sar-adc-bit1-no-flip.html).

Reply:
That's right.